recipe/meta.yaml

```diff
@@ -28,7 +28,7 @@ requirements:
     - typer
     - authlib
     - psycopg2
```

Comment:
this also seems to be an optional dependency, since quetz uses a slqlite database per default. i'd also suggest to remove it here.

Reply:
quetz already provides a useful error message here, so it's probably okay to remove it as a run requirement.

https://github.com/mamba-org/quetz/blob/9b344228458db6add87bc13c3ac368f874aa97fe/quetz/database_extensions.py